doc/source/serve/scaling-and-resource-allocation.md

```diff
@@ -0,0 +1,160 @@
+:::
+**max_replicas[default_value=1]**: Max replicas is the maximum number of replicas for the deployment. Ray Serve Autoscaling will rely on the Ray Autoscaler to scale up more nodes when the currently available cluster resources (CPUs, GPUs, etc) are not enough to bring up more replicas.
+
+**target_num_ongoing_requests_per_replica[default_value=1]**: The config is to maintain how many ongoing requests are expected to run concurrently per replica at most. If the number is lower, the scale up will be done more aggressively.
+:::{note}
+- It is always recommended to load test your workloads. For example, if the use case is latency sensitive, you can lower the `target_num_ongoing_requests_per_replica` number to maintain high performance.
+- Internally, the autoscaler will decide to scale up or down by comparing `target_num_ongoing_requests_per_replica` to the number of `RUNNING` and `PENDING` tasks on each replica.
+- `target_num_ongoing_requests_per_replica` is only a target value used for autoscaling (not a hard limit), the real ongoing requests number can be higher than the config.
+:::
+
+**downscale_delay_s[default_value=600.0]**: The config is to control how long the cluster needs to wait before scaling down replicas.
+
+**upscale_delay_s[default_value=30.0]**: The config is to control how long the cluster need to wait before scaling up replicas.
+
+:::{note}
+`downscale_delay_s` and `upscale_delay_s` are to control the frequency of doing autoscaling work. For example, if your use case takes a long time to do initialization work, you can increase `downscale_delay_s` to make the downscaling happen slowly.
+:::
+
+**smoothing_factor[default_value=1]**: The multiplicative factor to speedup/slowdown each autoscaling step. For example, when the use case has high traffic volume in short period of time, you can increase `smoothing_factor` to scale up the resource quickly.
+
+**metrics_interval_s[default_value=10]**: This is control how often each replica sends metrics to the autoscaler. (Normally you don't need to change this config.)
+
+(serve-cpus-gpus)=
+
+## Resource Management (CPUs, GPUs)
+
+You may want to specify a deployment's resource requirements, for example if a deployment requires a GPU.  To assign hardware resources per replica, you can pass resource requirements to
+`ray_actor_options`.
+By default, each replica requires one CPU.
+To learn about options to pass in, take a look at [Resources with Actors guide](actor-resource-guide).
+
+For example, to create a deployment where each replica uses a single GPU, you can do the
+following:
+
+```python
+@serve.deployment(ray_actor_options={"num_gpus": 1})
+def func(*args):
+    return do_something_with_my_gpu()
+```
+
+(serve-fractional-resources-guide)=
+
+### Fractional Resources
```

Comment:
Can we use the term "fractional GPUs" and "fractional CPUs" somewhere in this section?

That seems to be an industry term referring to the what we're describing. For example, a Google search for "fractional gpu model serving" brings up links like [this](https://www.run.ai/blog/reduce-cost-by-75-with-fractional-gpu-for-deep-learning-inference) and [this](https://docs.netapp.com/us-en/netapp-solutions/ai/osrunai_fractional_gpu_allocation_for_less_demanding_or_interactive_workloads.html).

Reply:
As a start, the current Getting Started guide has [a section that talks about fractional GPUs](https://github.com/ray-project/ray/blob/d809d748cf2ec2e8fc31dbb1256ac33dbca006d0/doc/source/serve/getting_started.md?plain=1#L192-L233). You could take language from there if it helps.

Reply:
Makes sense, I'll address this

Reply:
Added it in the header